src/lib/bootloader/language.rb

```diff
@@ -0,0 +1,19 @@
+require "cfa/base_model"
+require "cfa/augeas_parser"
+
+module Bootloader
+  # read only model to get system language from sysconfig
+  # Uses CFA base model with augeas parser
+  class Language < CFA::BaseModel
```

Comment:
I think `Bootloader::Language` is wrong. The class reads the systemwide language settings, there is nothing bootloader specific. We may not go as far as refactoring [Yast::Language](https://github.com/yast/yast-country/blob/e1c633c63eedc8894e0b79874dea9d2f9c54351b/language/src/modules/Language.rb#L553) but we should name this class right so that it can be simply moved to another package when it is needed there.

`Sysconfig::Language` ?